docs/cli_help.md

```diff
@@ -133,6 +133,23 @@
   * Applies to the most recent `--benchmark`, or all benchmarks if specified
     before any `--benchmark` arguments.
 
+* `--discard-on-throttle`
+  * Discard measurements if the GPU is throttled.
+  * Applies to the most recent `--benchmark`, or all benchmarks if specified
+    before any `--benchmark` arguments.
+
+* `--throttle-threshold <value>`
+  * Set the GPU throttle threshold as percentage of the peak clock rate.
+  * Default is 0.75 (75%).
+  * Applies to the most recent `--benchmark`, or all benchmarks if specified
+    before any `--benchmark` arguments.
+
+* `--throttle-recovery-delay <value>`
+  * Set the GPU throttle recovery delay in seconds.
+  * Default is 0.5 seconds.
```

Comment:
Suggestion: I think we could be more aggressive here. Just 0.05s was enough to get the frequency back up to a suitable rate for at least a few more samples in my testing (A6000, cub.select_if/cub.radix_sort), even with the threshold set to 90%.

My understanding is that as long as we're not constantly delaying for the same sample without making progress we can minimize this as much as possible.